tests/sentry/utils/performance_issues/test_performance_detection.py

```diff
@@ -980,6 +983,26 @@ def test_detects_consecutive_db_spans(self):
             ]
         )
 
+    def test_detects_file_io_main_thread(self):
```

Comment:
Would not be mad at a second test for the negative case! e.g., file IO _not_ on the main thread doesn't create a performance problem, or something similar